corehq/pillows/case.py

```diff
@@ -52,8 +61,42 @@ def calc_meta(cls):
         })
 
     def change_transform(self, doc_dict):
-        doc_ret = copy.deepcopy(doc_dict)
-        if not doc_ret.get("owner_id"):
-            if doc_ret.get("user_id"):
-                doc_ret["owner_id"] = doc_ret["user_id"]
-        return doc_ret
+        return transform_case_for_elasticsearch(doc_dict)
+
+
+def transform_case_for_elasticsearch(doc_dict):
+    doc_ret = copy.deepcopy(doc_dict)
+    if not doc_ret.get("owner_id"):
+        if doc_ret.get("user_id"):
+            doc_ret["owner_id"] = doc_ret["user_id"]
+    return doc_ret
+
+
+def prepare_sql_case_json_for_elasticsearch(sql_case_json):
+    prepped_case = transform_case_for_elasticsearch(sql_case_json)
+    # todo: these are required for consistency with couch representation, figure out how best to deal with it
+    prepped_case['doc_type'] = 'CommCareCase'
+    prepped_case['_id'] = prepped_case['case_id']
```

Comment:
wasn't sure what our long term strategy should be for this property coming out of elastic